CPT331.Web/Controllers/AccountController.cs

```diff
@@ -4,24 +4,24 @@
 using System.Web;
 using System.Web.Mvc;
 using CPT331.Web.Models.Account;
+using CPT331.Web.Attributes;
 
 namespace CPT331.Web.Controllers
 {
+    [AdminAuthorize]
     public class AccountController : Controller
     {
         // GET: Login
-        [AcceptVerbs(HttpVerbs.Get)]
+        [AllowAnonymous]
+        [HttpGet]
         public ActionResult Login()
         {
-            if (Session["user"] != null)
-            {                
-                return RedirectToAction("Home", "Admin");
-            }
             ViewBag.Title = "Admin";
             return View();
         }
 
-        [AcceptVerbs(HttpVerbs.Post)]
+        [AllowAnonymous]
+        [HttpPost]
         public ActionResult Login(string loginName, string password)
         {
             if ("administrator".Equals(loginName) && "Rgx53r$t5r".Equals(password))
```

Comment:
Yes, I agree. I also think in this context, I prefer to have the literal value on the right.

* ```loginName.Equals("administrator")``` is a nicer convention than ```"administrator".Equals(loginName)```

* And ```loginName == "administrator"``` is easier to understand ```loginName.Equals("administrator")```.

The use of Equals() is too Java for my liking :)